ccloud/docs/beginner-cloud.rst

```diff
@@ -637,73 +585,40 @@ Run kafka-connect-datagen connector with wildcard ACLs
       ../../ccloud/ccloud-generate-cp-configs.sh /tmp/client.config &>/dev/null
       source delta_configs/env.delta
 
-#. Run the provided :devx-examples:`docker-compose.yml file|ccloud/beginner-cloud/docker-compose.yml`
-   which is a |kconnect| container with the `kafka-connect-datagen <https://www.confluent.io/hub/confluentinc/kafka-connect-datagen>`__ plugin:
+#. Review the managed connector config file :devx-examples:`datagen_ccloud_pageviews.json|ccloud/beginner-cloud/datagen_ccloud_pageviews.json`:
 
-   .. code-block:: bash
+   .. literalinclude:: ../beginner-cloud/datagen_ccloud_pageviews.json
 
-      docker-compose up -d
-
-   Your output should resemble:
+#. Create a managed connector in Confluent Cloud with the provided configurations mentioned above using the following commands:
 
    .. code-block:: text
 
-      Creating network "beginner-cloud_default" with the default driver
-      Creating connect-cloud ... done
+      source ../../utils/ccloud_library.sh 
+      ccloud::create_connector datagen_ccloud_pageviews.json
```

Comment:
This is a library function which is great for automation (and the `start.sh` script) but for the docs it is more instructive to have the user learn the Confluent Cloud CLI command `ccloud connector create` instead.